apps/pie-docs/src/_includes/scripts.njk

```diff
@@ -5,3 +5,5 @@
 <script defer src="{{ '/assets/js/prism.js' | url }}" type="module"></script>
 <script defer src="{{ '/assets/js/awesome.js' | url }}" type="module"></script>
 <script defer src="{{ '/assets/js/scroll-to-top.js' | url }}" type="module"></script>
+<script defer src="{{ 'assets/js/search.js' | url }}" type="module"></script>
+<script src="https://cdn.jsdelivr.net/npm/@docsearch/js@3"></script>
```

Comment:
same as the CSS above, can we pass this through via node_modules as we do stuff like f-cookie-banner?